CHANGELOG.rst

```diff
@@ -2,6 +2,25 @@
 Vivisect Changelog
 ******************
 
+V1.2.1 - 2024-07-24
+===================
+Features
+--------
+- Update to large resource class in CI config.
```

Comment:
nit, but this should probably be moved down to fixes since it's not really a viv feature and this is a patch level release :)

Reply:
Feature felt a little closer than fix, but it's not really either of them I guess. So I just renamed it devops and added a label for that.